isis/src/base/objs/ShapeModel/ShapeModel.h

```diff
@@ -74,6 +77,11 @@ namespace Isis {
    *                            setSurfacePoint() & clearSurfacePoint() virtual
    *                            to give some hope of a consistent internal state
    *                            in derived models.
+   *   @history 2018-02-05 Cole Neubauer - added virtual method setTargetRadii to be 
+   *                            called from intersectEllipsoid. This is to avoid 
```

Comment:
https://isis.astrogeology.usgs.gov/documents/CodingStandards/CodingStandards.html#documentationFormatting
"When wrapping a Doxygen history entry, the subsequent lines should begin 4 spaces after the first name of the first programmer submitting the change."

If prior history entries are improperly indented, please fix them too.